books.txt

```diff
@@ -1,2 +1,2 @@
-First book
-Second book 
+First book author1
```

Comment:
I think that we should add a comma to distinguish the name of the book from the author, what do you think ?

Reply:
@spaenleh  more details please

Reply:
I was thinking we could have the format as follows:
```suggestion
First book, author1
```